app/services/weets.js

```diff
@@ -0,0 +1,11 @@
+const axios = require('axios');
+const { url } = require('../../config').common.api;
+
+exports.getWeet = async () => {
+  try {
+    const { data } = await axios.get(url);
+    return data;
+  } catch (error) {
+    return error;
```

Comment:
By standard, we do not return the same error that is generated when we call an external API or the DB and this fails, for that we use custom errors, to see the ones we already have in `app/errors.js`, in addition to this you should add the status code for said error in this `app/middlewares/errors.js` file otherwise a default status code will be used.

Reply:
What you can do is use a `logger.error` using the logger that is in the project and show the error returned by the external API or the DB.